src/qibolab/platform.py

```diff
@@ -352,6 +352,123 @@ def create_tii_rfsoc4x2(runcard, address=None):
+    # from qibolab.instruments.dummy_oscillator import (
+    #    DummyLocalOscillator as LocalOscillator,
+    # )
+    from qibolab.instruments.erasynth import ERA
+    from qibolab.instruments.rfsoc import RFSoC as TII_ZCU111
+
+    # Create channel objects
+    channels = ChannelMap()
+
+    # QUBIT 0 # D2
+    # readout
+    channels |= ChannelMap.from_names("L3-30_ro")  # dac6
+    # feedback
+    channels |= ChannelMap.from_names("L2-4-RO_0")  # adc0
+    # drive
+    channels |= ChannelMap.from_names("L4-29_qd")  # dac3
+    # Flux
+    channels |= ChannelMap.from_names("L1-22_fl")
+    # QUBIT 1 # D1
+    # readout
+    channels |= ChannelMap.from_names("L3-30_ro")  # dac6
+    # feedback
+    channels |= ChannelMap.from_names("L2-4-RO_1")  # adc1
+    # drive
+    channels |= ChannelMap.from_names("L4-30_qd")  # dac4
+    # Flux
+    channels |= ChannelMap.from_names("L1-23_fl")  # dac0
+    # QUBIT 2 # D5
+    # readout
+    channels |= ChannelMap.from_names("L3-30_ro")
+    # feedback
+    channels |= ChannelMap.from_names("L2-4-RO_2")
+    # drive
+    channels |= ChannelMap.from_names("L4-31_qd")
+    # Flux
+    channels |= ChannelMap.from_names("L1-24_fl")
+
+    # Map controllers to qubit channels (HARDCODED)
+    # Qubit 0
+    # readout
+    channels["L3-30_ro"].ports = [("dac6", 6)]
+    # feedback
+    channels["L2-4-RO_0"].ports = [("adc0", 0)]
+    # drive
+    channels["L4-29_qd"].ports = [("dac3", 3)]
+    # Flux
+    channels["L1-22_fl"].ports = [("dac0", 0)]
+    # Qubit 1
+    # Readout
+    channels["L3-30_ro"].ports = [("dac3", 6)]
+    # feedback
+    channels["L2-4-RO_1"].ports = [("adc1", 1)]
+    # drive
+    channels["L4-30_qd"].ports = [("dac4", 4)]
+    # Flux
+    channels["L1-23_fl"].ports = [("dac1", 1)]
+    # Qubit +2
+    # Readout
+    channels["L3-30_ro"].ports = [("dac6", 6)]
+    # feedback
+    channels["L2-4-RO_2"].ports = [("adc2", 2)]
+    # drive
+    channels["L4-31_qd"].ports = [("dac5", 5)]
+    # Flux
+    channels["L1-24_fl"].ports = [("dac2", 2)]
+
+    # Instantiate local oscillators (HARDCODED)
+    local_oscillators = [
+        ERA("ErasynthLO", "192.168.0.212", ethernet=True),
+    ]
+
+    # Instantiate QICK instruments
+    if address is None:
+        address = "192.168.0.81:6000"
+    controller = TII_ZCU111("tii_zcu111", address)
+
+    # Readout local oscillator
+    local_oscillators[0].frequency = 7_500_000_000
+    local_oscillators[0].power = 10
+    channels["L3-30_ro"].local_oscillator = local_oscillators[0]
+
+    instruments = [controller] + local_oscillators
+
+    design = InstrumentDesign(instruments, channels)
+    platform = DesignPlatform("tii_rfsocZCU111", design, runcard)
+
+    # assign channels to qubits
+    qubits = platform.qubits
+    qubits[0].readout = channels["L3-30_ro"]
+    qubits[0].feedback = channels["L2-4-RO_0"]
+    qubits[0].drive = channels["L4-29_qd"]
+    qubits[0].flux = channels["L1-22_fl"]
+    channels["L1-22_fl"].qubit = qubits[0]
```

Comment:
@stavros11 what did we decide in the end for these `.qubit` attributes? Or are we still lacking an actual decision?

Reply:
No decision yet. For sure in #438 we will not change anything related to this so it is fine to leave as it is in this PR.

I will see what can be simplified during the second refactor (removal of `MultiqubitPlatform`). The main issue is the cyclic dependency `Qubit -> Channel -> Qubit`. I believe the second part (`Channel -> Qubit`) can be dropped. For the first part (`Qubit -> Channel`) I do not see any issue keeping, but maybe we can check again once we have a cleaner `Platform`.